src/app/problems/problem-view/submission-snippet/submission-snippet.component.html

```diff
@@ -24,7 +24,7 @@ <h2 class="tui-text_h6 tui-space_top-0 previous-submissions-island__title">Past
                                 / {{submission.token_value.toFixed(2) }}</p>
                             <p class="tui-island__paragraph">Answer:</p>
                             <ng-container *ngIf="submission.show_answer else noAnswerShown">
-                                <div *ngFor="let safeAnswer of submission.safeAnswer"
+                                <div *ngFor="let safeAnswer of submission.answer_display"
```

Comment:
This will make the website vulnerable to XSS attack. Let's find a better solution.